members/SWYA766_702158.yaml

```diff
@@ -0,0 +1,38 @@
+bioguide: SWYA766_702158
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "https://docs.google.com/forms/viewform?bc=transparent&embedded=true&f=Arial%252C%2BVerdana%252C%2Bsans-serif&hl=en&htc=%2523666666&id=161LC0kc7zjfYxt1JzsunN1aL7nd1k-gwK8BUrnUB3yk&lci"
+    - fill_in:
+        - name: Name Full
+          selector: input#entry_1835936712
+          value: $NAME_FULL
+          required: true
+        - name: email
+          selector: input#entry_1036482007
+          value: $EMAIL
+          required: true
+        - name: Subject
+          selector: input#entry_115642149
+          value: $EMAIL
+          required: true
+        - name: Message
+          selector: textarea#entry_1204170891
+          value: $MESSAGE
+          required: true
+    - select:
+        - name: Division
+          selector: select#entry_983581508
+          value: Attorney General
+          required: true
+    - click_on:
+        - value: Submit
+          selector: input[type='submit'][value='Submit']
+    - wait:
+        - value: 3
```

Comment:
Is it possible to use a `find` instead? That will wait a maximum of 5 seconds but will take less time if it finds it sooner